### PR TITLE
feat: implement validator detail page

### PR DESCRIPTION
**Pull request by sunnygleason**
Problem:

* need implementation of validator detail

Solution:

* implement validator detail page in v2

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -4,12 +4,12 @@ import {BrowserRouter} from 'react-router-dom';
 
 import './index.css';
 import App from './App';
-import AppV2 from './AppV2';
+// import AppV2 from './AppV2';
```
Comment (sunnygleason) [inline src/index.js]: (temporarily exclude AppV2 until JS is fully merged)

File: src/App.js [hunk 2]
```diff
@@ -708,6 +712,16 @@ class App extends Component {
                   <Bx2PanelValidatorsThemed nodes={this.state.nodes} />
                 )}
               />
+              <Route
```
Comment (sunnygleason) [inline src/App.js]: validator detail page takes list of nodes already in state & specific id from path

File: src/index.js (hunk 1 above)
Comment (mvines) [reply src/index.js]: Why do we need this?  If the v2/ routes are a little busted that's fine by me.

Comment (sunnygleason) [reply src/index.js]: AppV2 is @manuel-calavera rewrite, which needs updating & should hopefully land soon

**Comment by sunnygleason** [comment]
@mvines ty!

**Pull request merged**